docs/pages/pmd/userdocs/pmd_report_formats.md

```diff
@@ -2,7 +2,7 @@
 title: Report formats for PMD
 tags: [pmd, userdocs]
 keywords: [formats, renderers]
-summary: "Overview of the built-in report formats for PMD"
+summary: 'Overview of the built-in report formats for PMD'
```

Comment:
I'd prefer to not do unrelated changes...

Reply:
Also Prettier formatting. Next time I'll disable it